### DISCUSSION TITLE
Button on screen to 'reboot/restart'

### DISCUSSION THREAD

**Discussion by rjblake** [HASwitchPlate/openHASP#639] [Q&A] [answered]
Every now and then I find I need to reboot my display. Would be good to be able to be able to do this from the display, but I cannot for the life of me work out how to do it. I've tried adding a button with an action 'reboot' but it does nothing. I'm sure I've seen an example somewhere but cannot find it. Any tips/sample code?

**Comment by fvanroie** [comment, marked as answer]
Try: 

```json
{"id":10,"obj":"btn","x":20,"y":40,"w":90,"h":75,"text":"10%","action":{"up":"reboot"}}
```

**Reply by rjblake** [reply]
Awesome! Thanks so much, works perfectly. I had missed the "up" in the action. Still getting my head around things :)
